oem/ibm/libpldmresponder/file_io_type_dump.cpp

```diff
@@ -438,8 +438,8 @@ void DumpHandler::readIntoMemory(uint32_t offset, uint32_t length,
                                  sdeventplus::Event& event)
 {
     auto path = findDumpObjPath(fileHandle);
-    static constexpr auto dumpFilepathInterface =
-        "xyz.openbmc_project.Common.FilePath";
+    // static constexpr auto dumpFilepathInterface =
```

Comment:
Do we need to keep this commented code ?

Reply:
No, removed it.